Cargo.toml

```diff
@@ -17,6 +17,7 @@ object = "0.20"
 gimli = "0.22.0"
 capstone = "0.7.0"
 addr2line = "0.13.0"
+syntect = "4.4.0"
```

Comment:
For now, this is used only by `examples/repl.rs`. I think it'd be good to move this to the `[dev-dependencies]` section below. However, the `Snippet` helper is in the main source tree and it might be helpful for other use cases too, so we can't easily make this a dev-dependency.

Maybe we can use a feature flag instead? So `mod pretty` can be compiled conditionally with `#[cfg(feature = "syntax-hughlighting")]` and the dependency can be included as `syntect = { version = "4.4.0", optional = true }`. What do you think?

Reply:
`syntect` is used in `headcrab::symbol::source::pretty`, which doesn't have access to dev dependencies. I agree that a feature flag would be nice.

Reply:
@nbaksalyar I have added the feature gates in the latest revision. But now we need to run the repl example as
`cargo run --example repl --features syntax-highlighting`. Is that okay ?

Reply:
Also, if the feature is not enabled, the examples should still work